conf.yaml

```diff
@@ -878,7 +878,7 @@ contents:
             index:      app-search-docs/index.asciidoc
             private:    1
             current:    *stackcurrent
-            branches:   [ 7.7 ]
+            branches:   [ 7.8, 7.7 ]
             chunk:     1
             tags:       App Search/Guide
             subject:    App Search
```

Comment:
@chriscressman - can you check this is correct for Enterprise Search? Thanks

Reply:
Yes, looks good :rocket: